packages/typescript-eslint-parser/src/parser.ts

```diff
@@ -0,0 +1,67 @@
+import traverser from 'eslint/lib/util/traverser';
+import * as typescriptESTree from 'typescript-estree';
+import { analyzeScope } from './analyze-scope';
+import { ParserOptions } from './parser-options';
+import { visitorKeys } from './visitor-keys';
+
+const packageJSON = require('../package.json');
```

Comment:
Just for my own understanding, why does require need to be used for JSON files? Is it just to avoid having to declare a JSON module interface, or is there something else at play here?

Reply:
"Is it just to avoid having to declare a JSON module interface", yes

Reply:
In this case, if you enabled `resolveJsonModule` in TS to allow you to import it, it will work, but because the package.json lives in the directory above the `src` it changes the output directory (dist) to contain nesting:

dist/
  package.json
  src/

Instead of the 1:1 mapping with src